src/inmanta/server/services/resourceservice.py

```diff
@@ -138,6 +134,7 @@ def __init__(self) -> None:
         ] = {}
         # lock to ensure only one inflight request
         self._increment_cache_locks: dict[uuid.UUID, asyncio.Lock] = defaultdict(lambda: asyncio.Lock())
+        self._parent_resource_action_logger = logging.getLogger(const.NAME_RESOURCE_ACTION_LOGGER)
```

Comment:
I find the word "parent" in this context a bit out of place. I would prefer plainly `_resource_action_logger`, or `_resource_action_root_logger` if you want to make it explicit.

Not a strict change request.